### PR TITLE
Remove support for supplementary properties from SchemaElements 

### PR DESCRIPTION
**Pull request by mandy-chessell**
Signed-off-by: Mandy Chessell <mandy.e.chessell@gmail.com>

## Description

[Supplementary properties](https://egeria-project.org/types/3/0395-Supplementary-Properties/)  are used to add additional descriptive properties to technical metadata elements.

This capability is supported by Asset Manager OMAS, Stewardship Action OMAS, Governance Program OMAS.  It is soon to be added to Asset Owner OMAS and Asset Consumer OMAS (via OCF) to allow the retrieval of ths information to owners and consumers.

Asset Manager OMAS supported this capability for Assets, SchemaElements, Endpoints and SoftwareCapabilities.  This PR removes supplementary properties support for SchemaElements because the direct usage of the SchemaElements is typically done by technical users.  Business users typically do not identify with a specific schema element.  They would more likely be using glossary terms linked with Semantic Assignment to retrieve collections of schema elements that match their business understanding.  Therefore the overhead of supporting business descriptions for schema elements is not justified.

## Related Issue(s)

* Issue covering the development of - #3562
* Issue covering the discussion of why removing supplementary properties from schema elements - #7230

## Testing

FVT testing as much as it is.  Asset Manager OMAS is still in its initial development phase and so the FVTs do not cover all of the features.

## Release Notes & Documentation

There is a matching PR on egeria-docs to remove schema elements from the description of supplementary properties.

### CODE REVIEW

File: open-metadata-implementation/access-services/asset-manager/asset-manager-api/src/main/java/org/odpi/openmetadata/accessservices/assetmanager/properties/SchemaTypeProperties.java [hunk 1]
```diff
@@ -111,6 +113,28 @@ public void setAuthor(String author)
     }
 
 
+    /**
+     * Return details of the expected usage of this technical element.
+     *
+     * @return string text
+     */
+    public String getUsage()
```
Comment (davidradl) [inline open-metadata-implementation/access-services/asset-manager/asset-manager-api/src/main/java/org/odpi/openmetadata/accessservices/assetmanager/properties/SchemaTypeProperties.java:121]: I am curious about the thinking on why usage is in the technical metadata.

**Review by davidradl — APPROVED** [review]

File: open-metadata-implementation/access-services/asset-manager/asset-manager-api/src/main/java/org/odpi/openmetadata/accessservices/assetmanager/properties/SchemaTypeProperties.java (hunk 1 above)
Comment (mandy-chessell) [reply open-metadata-implementation/access-services/asset-manager/asset-manager-api/src/main/java/org/odpi/openmetadata/accessservices/assetmanager/properties/SchemaTypeProperties.java:121]: Similar answer as to why displayName and description is in the technical metadata - it is possible that the tool is able to scavenge usage information from comments or similar descriptive information.

Comment (mandy-chessell) [reply open-metadata-implementation/access-services/asset-manager/asset-manager-api/src/main/java/org/odpi/openmetadata/accessservices/assetmanager/properties/SchemaTypeProperties.java:121]: This comment should really be addressed to the open metadata types - here it is in the documentation.  https://egeria-project.org/types/5/0501-Schema-Elements/

If it is not answering your questions you should make an update to the documentation when you find out the answers.

**Comment by mandy-chessell** [comment]
This is the PR to remove SchemaElements from the description of supplementary properties 

https://github.com/odpi/egeria-docs/pull/627

**Pull request merged**